recipes/ceres-solver/meta.yaml

```
@@ -0,0 +1,53 @@
+{% set version = "1.14.0" %}
+
+package:
+  name: ceres-solver
+  version: {{ version }}
+
+source:
+  url: http://ceres-solver.org/ceres-solver-{{ version }}.tar.gz
+  sha256: 4744005fc3b902fed886ea418df70690caa8e2ff6b5a90f3dd88a3d291ef8e8e
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+
+requirements:
+  build:
+    - cmake
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+
+  host:
+    - glog
+    - gflags
+    - suitesparse
+    - tbb
+    - eigen
+
+  run:
+    - glog
+    - gflags
+    - suitesparse
+    - tbb
+    - eigen
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/ceres/ceres.h  # [unix]
+    - test -f ${PREFIX}/lib/libceres${SHLIB_EXT}  # [unix]
+    - if not exist %PREFIX%\\Library\\include\\ceres\\ceres.h exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\lib\\ceres.lib exit 1  # [win]
+    - if not exist %PREFIX%\\Library\\bin\\ceres.dll exit 1  # [win]
+    - conda inspect linkages -p ${PREFIX} ceres-solver  # [not win]
+    - conda inspect objects -p ${PREFIX} ceres-solver  # [osx]
+
+about:
+  home: http://ceres-solver.org
+  license: BSD 3 Clause
```

From https://opensource.org/licenses/alphabetical:
```suggestion
  license: BSD-3-Clause
```